VirtualFinland.UserAPI/src/VirtualFinland.UsersAPI/Activities/CodeSets/Operations/GetAllOccupations.cs

```diff
@@ -30,13 +30,14 @@ public async Task<List<Occupation>> Handle(Query request, CancellationToken canc
                         o.Name?.Swedish),
                     new LanguageTranslations(o.Description?.Finland,
                         o.Description?.English,
-                        o.Description?.Swedish)))
+                        o.Description?.Swedish),
+                    o.HierarchyLevel))
                 .OrderBy(o=> int.Parse(o.Id!)).ToList() ?? new List<Occupation>();
```

Comment:
Luulen että tuon listan hierarkia-puun rakentamiseen tarvitsisi myös sen alkuperäisen järjestyksen mikä json-datassa näyttäis tulevan kentästä `order`, ja tämä orderBy pitäisi kanssa sitten tehdä id:n sijasta order-kenttää katsoen.

Reply:
Ah, toi "Id" tässä kohtaa olekaan se guid-tyyppinen tieto vaan tuollainen koodi "01", "02" eli joo niillä järjestys meneekin ihan oikein.